### PR TITLE
[#1503] Cypress test to end a conversation in chatplugin

### LINKED ISSUE #1503: Cypress test to end a current conversation and create an entirely new conversation in the chatplugin
This test should end a current conversation in the chatplugin and create a new conversation with the same channelid

### PR DESCRIPTION
**Pull request by ogbeche77**
Fixes #1503

### CODE REVIEW

File: integration/cypress.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
        "searchQuery": "Cypress Filter",
        "messageInbox": "Hello from Cypress Inbox!",
        "messageChatplugin": "Hello from Cypress ChatPlugin!",
+       "newNessageChatplugin": "Hello once again from Cypress ChatPlugin!",
```
Comment (pascal-airy) [inline integration/cypress.json:13]: is this a new"N"essage ? :)

File: integration/chat-plugin/end_conversation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  cyBubble,
+  cyInputbarTextarea,
+  cyInputbarButton,
+  cyChatPluginMessageList,
+  cyChatPluginHeaderBarCloseButton,
+  cyChatPluginEndChatModalButton,
+  cyChatPluginStartNewConversation,
+} from 'chat-plugin-handles';
+
+describe('Websocket test', () => {
```
Comment (pascal-airy) [inline integration/chat-plugin/end_conversation.spec.ts:11]: Old description

File: integration/ui/send_chatplugin_message.spec.ts [hunk 3]
```diff
@@ -36,7 +36,7 @@ describe('Send chat plugin message', () => {
     cy.wait(500);
     cy.get(`[data-cy=${cyChannelsChatPluginList}]`).filter(`:contains("${Cypress.env('chatPluginName')}")`);
 
-    cy.visit('http://airy.core/chatplugin/ui/example?channel_id=' + Cypress.env('channelId'));
+    cy.visit('/chatplugin/ui/example?channel_id=' + Cypress.env('channelId'));
```
Comment (pascal-airy) [inline integration/ui/send_chatplugin_message.spec.ts:39]: <3

File: integration/cypress.json (hunk 1 above)
Comment (ogbeche77) [reply integration/cypress.json:13]: Yea, that was typo

**Review by pascal-airy — APPROVED** [review]

**Review by pascal-airy — APPROVED** [review]

**Pull request merged**